chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
Most of these are declared in if statements, which would make them fall out of block scope. Overall, common consensus between using `var` or `let` is not reached within the codebase, I feel. @dstillman prefers var for function scope, I myself would prefer to either sticking with one or the other, without mixing.